Server/Test/v_entryTests.js

```diff
@@ -115,12 +169,22 @@ describe('Get all entries', () => {
         chai.request(app).get('/api/v1/entries')
             .set('Authorization', testEntry[8].token)
             .end((err, res) => {
-                expect(res).to.have.status(401);
+                expect(res).to.have.status(400);
```

Comment:
This should not be a 400, you need to do more research on the status codes. Change to the right status code after wards